docs/src/reference.md

```diff
@@ -30,24 +28,25 @@ IRTools.isreturn
 IRTools.returnvalue
 IRTools.returntype
 IRTools.canbranch
-IRTools.basicblock
 IRTools.explicitbranch!
-IRTools.returnvalue
 IRTools.argument!
 IRTools.emptyargs!
 IRTools.deletearg!
 IRTools.block!
 IRTools.deleteblock!
 IRTools.block
 IRTools.blocks
+IRTools.branch!
+IRTools.return!
 IRTools.successors
 IRTools.predecessors
+keys
+haskey
 push!
 pushfirst!
 insert!
 IRTools.insertafter!
+permute!
 empty
-keys
-haskey
 IRTools.Pipe
```

Comment:
`Pipe` is so useful that it's worth to be listed in an upper position, imho.

Reply:
It probably deserves its own section in the docs, rather than being buried in the reference at all.

Reply:
> It probably deserves its own section in the docs, rather than being buried in the reference at all.

yeah, sounds fair. I may try to write it down once I get used to this package.